docs/SDD/02.overview/00.overview.adoc

```diff
@@ -94,9 +94,10 @@ The endpoints used for SCIM are:
 The Resource API Endpoints offered by the PEP component are protected based on the unique identifier of the Resource Owner that is adding/removing/editing resources.
 The Resource API is protected with OAuth/OIDC in the PEP, expecting any of these tokens:
 
-* JWT id_tokens: in this case the PEP extracts the necessary claims from the JWT uniquely identifying the user (“sub” parameter) and the signature of the JWT will also be verified.
-
-* RPT Token: in this case the PEP extracts the necessary claims from the RPT uniquely from the "pct_claims" parameter
+* JWT id_tokens: in this case the PEP extracts the necessary claims from the JWT uniquely identifying the user (“sub” parameter).
+The signature of this token will be verified if the signature verification is enabled in the environment variables.
+In case it is enabled, it will be distinguished if the JWT obtained in the header is signed with the internal keys of the platform or the building block.
```

Comment:
Perhaps also add here the behavior in case the signature verification is disabled in the variables?